hermes-console-vue/src/views/admin/readiness/ReadinessView.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+  import { useI18n } from 'vue-i18n';
+  import { useReadiness } from '@/composables/use-readiness/useReadiness';
+  import ConsoleAlert from '@/components/console-alert/ConsoleAlert.vue';
+  import LoadingSpinner from '@/components/loading-spinner/LoadingSpinner.vue';
+  import ReadinessBreadcrumbs from '@/views/admin/readiness/readiness-breadcrumbs/ReadinessBreadcrumbs.vue';
+
+  const { t } = useI18n();
+
+  const { datacentersReadiness, loading, error } = useReadiness();
+</script>
+
+<template>
+  <v-container fill-height fluid class="mx-auto">
+    <v-row dense>
+      <v-col md="12">
+        <readiness-breadcrumbs></readiness-breadcrumbs>
+        <loading-spinner v-if="loading" />
+        <console-alert
+          v-if="error"
+          :title="t('readiness.connectionError.title')"
+          :text="t('readiness.connectionError.text')"
+          type="error"
+        />
+      </v-col>
+    </v-row>
+    <v-container v-if="!loading && !error">
+      <v-row dense>
+        <v-col md="12">
+          <p class="text-h4 font-weight-bold mb-3">
+            {{ t('readiness.title') }}
+          </p>
+        </v-col>
+      </v-row>
+      <v-row dense>
+        <v-col md="12">
+          <v-table>
+            <thead>
+              <tr>
+                <th class="text-left">{{ t('readiness.index') }}</th>
+                <th class="text-left">{{ t('readiness.datacenter') }}</th>
+                <th class="text-left">{{ t('readiness.isReady') }}</th>
+                <th class="text-left">{{ t('readiness.control') }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(item, index) in datacentersReadiness" :key="index">
+                <td>{{ index + 1 }}</td>
+                <td>{{ item.datacenter }}</td>
+                <td>{{ item.isReady }}</td>
+                <td class="w-0">
+                  <v-btn
```

Comment:
wdyt about using `v-btn` in `text` or `plain` variant for action buttons in tables? (haven't try it yet – just wondering if this would make those buttons outstand less from the rest of tables content 🤔)

Reply:
Looks cool, changed 
![image](https://user-images.githubusercontent.com/76775507/230072383-b4520080-fea5-455f-911a-05af1989d9f3.png)
![image](https://user-images.githubusercontent.com/76775507/230072437-dcf1588a-faac-4a94-88fc-350e918f1a3f.png)